server/fleet/hosts.go

```diff
@@ -119,29 +137,32 @@ type HostListOptions struct {
 	// populated.
 	AdditionalFilters []string
 	// StatusFilter selects the online status of the hosts.
-	StatusFilter HostStatus
+	StatusFilter HostStatus `json:"status"`
```

Comment:
Double checking: Will this break other endpoints that use `fleet.HostListOptions`?

Reply:
(See `NOTE` in `HostListOptions` definition doc.)

Reply:
good news is that tests are passing, but i think still worth it to do some extra sanity tests